src/sagas/prescriptions.js

```diff
@@ -68,8 +68,11 @@ function* onFetchPrescription(action) {
 function* onCreatePrescription(action) {
   try {
     const prescription = yield call(prescriptionsService.createPrescription, action.payload);
+    console.log(prescription);
+    //server return prescription with prescribedBy, prescribedTo and medicines like ID, thats why we can`t save the prescription direct in all prescription in redux
```

Comment:
Use the prescription Id, if available, to dispatch a new action that triggeres a server call to fetch the whole prescription and then set it in the state